static/app/utils/theme.tsx

```diff
@@ -808,6 +808,8 @@ const commonTheme = {
     collapsedWidth: '70px',
     semiCollapsedWidth: '100px',
     expandedWidth: '220px',
+    v2_width: '80px',
+    v2_panelWidth: '160px',
```

Comment:
Slight adjustments from the spec'd designs to accommodate font rendering differences